spk/filebrowser/Makefile

```
@@ -1,6 +1,6 @@
 SPK_NAME = filebrowser
-SPK_VERS = 2.18.0
-SPK_REV = 2
+SPK_REV = 1
+SPK_VERS = 2.20.1
```

any reason to change the order of those?
```suggestion
SPK_VERS = 2.20.1
SPK_REV = 1
```

---

I have no idea how that happened. (magic keyboard shortcut?) Thanks for pointing that out.